examples/RNN_toy_example.py

```diff
@@ -0,0 +1,142 @@
+import numpy as np
```

Comment:
Please do not use `import something as st`, this is a part of Blocks code style.

You can find the list of whole restrictions like this one in http://blocks.readthedocs.org/en/latest/development/index.html